index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header class="mobile-header">
+        <p>Diego Hernández</p>
+        <button type="button" class="menu"> </button>
+   
+    </header>
+
+
+    <div class="content"> 
+        <img src="./img1.png" alt="background"> 
+        <a> <img src="./smedia.png" alt="dsds"> </a>
```

Comment:
- [ ] Please create individual elements for each one of the social media tags, the icons you can also find those icons in [Font Awesome](https://fontawesome.com/icons), create the social media icons according as the requirements says
> The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag)